src/QuickMarcEditor/QuickMarcDeriveWrapper.js

```diff
@@ -119,8 +119,6 @@ const QuickMarcDeriveWrapper = ({
 
     return mutator.quickMarcEditMarcRecord.POST(formValuesForDerive)
       .then(async ({ qmRecordId }) => {
-        onClose('id'); // https://issues.folio.org/browse/UIQM-82
```

Comment:
This functionality initially was added in [UIQM-82](https://issues.folio.org/browse/UIQM-82). I think the PO should be informed about this change.

`onClose('id')` was added to remove duplication and have one [source](https://github.com/folio-org/ui-inventory/blob/3d7c60326853a780e7bc16a95b6f84ea51a76d2b/src/routes/QuickMarcRoute.js#L13) of truth in `ui-inventory`.

Reply:
It looks like the third pane should display a loading indicator, until PUT returns a response ("Show an empty third pane with a loading indicator").

Reply:
You're correct, I've reverted that change and fixed not returned Promise in `saveLinksToNewRecord` function